_data/site.yaml

```diff
@@ -80,24 +88,24 @@ catalog_filters:
 resource_quick_links:
   - title: Data Management & Governance
     url: https://resources.data.gov/categories/data-management-governance/
-  - title: Guidance
-    url: https://resources.data.gov/categories/guidance/
   - title: Data Tools
     url: https://resources.data.gov/categories/data-tools/
-  - title: Case Studies & Examples
-    url: https://resources.data.gov/categories/case-studies-examples/
   - title: Data Inclubator
     url: https://resources.data.gov/categories/data-incubator/
+  - title: Guidance
+    url: https://resources.data.gov/categories/guidance/
+  - title: Case Studies & Examples
+    url: https://resources.data.gov/categories/case-studies-examples/
   - title: Skills Development
     url: https://resources.data.gov/categories/skills-development/
 
 strategy_quick_links:
   - title: 2021 Action Plan
     url: https://strategy.data.gov/2021/action-plan/
-  - title: 2021 Action Progress
-    url: https://strategy.data.gov/2021/progress/
   - title: 2020 Action Plan
     url: https://strategy.data.gov/2020/action-plan/
+  - title: 2021 Action Progress
+    url: https://strategy.data.gov/2021/progress/
   - title: 2020 Action Progress
     url: https://strategy.data.gov/2020/progress/
```

Comment:
I think it makes more sense to have it grouped by year (chronologically) and not progress vs. plan...

Reply:
Just following the comps that Carolina made. Let's get this in as a first pass then record thoughts like this on the QA ticket: https://github.com/GSA/data.gov/issues/4213